Disable defaulting of GCRYPT_LDFLAGS to -L/usr/lib

Prevents "unsafe for cross compilation" warnings that cause
do_qa_configure to fail.

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Paul Eggleton <paul.eggleton@linux.intel.com>

diff --git a/configure.in b/configure.in
index 98395ed..81c3a2c 100644
--- a/configure.in
+++ b/configure.in
@@ -1777,11 +1777,11 @@ then
 		GCRYPT_CPPFLAGS=`"$with_libgcrypt_config" --cflags 2>/dev/null`
 	fi
 
-	if test "x$GCRYPT_LDFLAGS" = "x"
-	then
-		gcrypt_exec_prefix=`"$with_libgcrypt_config" --exec-prefix 2>/dev/null`
-		GCRYPT_LDFLAGS="-L$gcrypt_exec_prefix/lib"
-	fi
+#	if test "x$GCRYPT_LDFLAGS" = "x"
+#	then
+#		gcrypt_exec_prefix=`"$with_libgcrypt_config" --exec-prefix 2>/dev/null`
+#		GCRYPT_LDFLAGS="-L$gcrypt_exec_prefix/lib"
+#	fi
 
 	if test "x$GCRYPT_LIBS" = "x"
 	then
